hbp_knowledge/receptors/dani2015.bel

```diff
@@ -112,52 +124,67 @@ Galzi, Devillers-Thiery, Bertrand, & Changeux, 1993b; Dani & Bertrand,
 2007; Decker & Dani, 1990; Gray, Rajan, Radcliffe, Yakehiro, & Dani,
 1996; McGehee, Heath, Gelber, Devay, & Role, 1995; Vernino et al.,
 1992)."
+SET Confidence = "High"
 act(p(FPLX:CHRN)) -> bp(GO:"membrane depolarization")
 tloc(a(MESH:"Cations, Divalent"), MESH:"Extracellular Space", MESH:"Intracellular Space") -> bp(GO:"membrane depolarization")
 tloc(a(CHEBI:"calcium(2+)"), MESH:"Extracellular Space", MESH:"Intracellular Space") -> bp(GO:"membrane depolarization")
+UNSET Confidence
 
 SET Evidence = "Although calcium modulation can act intracellularly,
 nAChRs also are allosterically modulated by extracellular calcium, leading
 to dramatic changes in the channel opening probability (Amador & Dani,
 1995; Mulle, Lena, & Changeux, 1992; Vernino et al., 1992)."
+SET Confidence = "High"
 SET MeSHAnatomy = "Extracellular Space"
 a(CHEBI:"calcium(2+)") reg act(p(FPLX:CHRN))
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = " Although
 many areas of the brain participate, nicotinic receptors of the midbrain dopa-
 mine (DA) area are particularly important during the initiation of the addic-
 tion process (Dani et al., 2014; De Biasi & Dani, 2011)."
+SET Confidence = "High"
 SET Cell = "midbrain dopaminergic neuron"
 #path(MESH:"Tobacco Smoking") -> act(p(FPLX:CHRN))
 act(p(FPLX:CHRN)) -> path(MESH:"Behavior, Addictive")
 UNSET Cell
+UNSET Confidence
 
 SET Evidence = " Activation of the presynaptic
 nAChRs (commonly but not exclusively α 7 * nAChRs) enhances the release
 of glutamate (Dani et al., 2000; Mansvelder & McGehee, 2000, 2002)."
+SET Confidence = "High"
 SET MeSHAnatomy = "Presynaptic Terminals"
 act(p(FPLX:CHRN)) -> sec(a(CHEBI:"glutamate(2-)"))
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "Simultaneously, activity of postsynaptic (and somatic) α 4 β 2 * nAChRs
 depolarizes DA neurons leading to enhanced action potential firing
 (Zhang et al., 2009)."
+SET Confidence = "High"
 SET Cell = "dopaminergic neuron"
 SET MeSHAnatomy = "Post-Synaptic Density"
 act(p(HBP:"alpha-4 beta-2 nAChR")) -> bp(GO:"depolarization of postsynaptic membrane")
 bp(GO:"depolarization of postsynaptic membrane") -> bp(GO:"regulation of action potential firing rate")
 UNSET MeSHAnatomy
 UNSET Cell
+UNSET Confidence
 
 SET Evidence = " This depolarization and firing of the DA neurons helps
 to relieve the divalent cation block of NMDA receptors and, thus, enables
 the NMDA receptors to participate in long-term synaptic potentiation of
 glutamatergic afferents onto midbrain dopamine neurons."
+SET Confidence = "Low"
 SET Cell = "dopaminergic neuron"
-#bp(GO:"depolarization of postsynaptic membrane") -> bp(GO:"NMDA glutamate receptor activity")
+#bp(GO:"depolarization of postsynaptic membrane") -| a(MESH:"Cations, Divalent")
+#a(MESH:"Cations, Divalent") -| bp(GO:"NMDA glutamate receptor activity")
+UNSET Cell
+#the error states that "GO:NMDA glutamate receptor activity should not be coded as an entity". How else can this be stated?
```

Comment:
you should only code the biologicl processes of GO as entity.
In this case i would instead code this as the activity of the receptor:
e.g. act(p(HGNCGENEFAMILY:"Glutamate ionotropic receptor NMDA type subunits"))

(maybe there's a better namespace for this receptor, i have only checked hgnc...)